src/main/java/com/epam/edumanagementsystem/util/imageUtil/rest/service/impl/ImageServiceImpl.java

```diff
@@ -35,7 +35,7 @@ public String saveImage(MultipartFile file) {
             try {
                 file.transferTo(new File(upload + File.separator + picUrl));
             } catch (IOException e) {
-                e.printStackTrace();
+               logger.error(e.getMessage());
```

Comment:
No need to log an error message and do nothing because, in the end, we will have a situation where the image won't get saved, and we will return a positive status code as if it succeeded. Instead, do not catch the exception. But if you are catching then after logging rethrow the exception.